.github/workflows/build.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
```

Comment:
Maybe `Arma`?

Reply:
Don't think this conveys the message at all.